features/registry/registry.feature

```diff
@@ -173,3 +173,68 @@ Feature: Testing registry
       | Mirroring completed in |
     Given the "myimage" image stream was created
     And the "myimage" image stream becomes ready
+
+  # @author xiuwang@redhat.com
+  # @case_id OCP-29693
+  @admin
+  Scenario: [Disconnect]Import image from a secure registry using node credentials
+    Given I have a project
+    And evaluation of `image_content_source_policy('image-policy-aosqe').mirror_registry(cached: false, user: admin)` is stored in the :mirror_registry clipboard
+    When I run the :tag client command with:
+      | source           | <%= cb.mirror_registry %>rhscl/mysql-80-rhel7:latest  | 
+      | dest             | mysql:8.0                                             |
+      | reference_policy | local                                                 |
+    Then the step should succeed
+    When I run the :new_app client command with:
+      | template | mysql-persistent              |
+      | p        | NAMESPACE=<%= project.name %> |
+    Then the step should succeed
+    When a pod becomes ready with labels:
+      | deployment=mysql-1 |
+    When I run the :describe client command with:
+      | resource | pod                | 
+      | l        | deployment=mysql-1 |
+    And the output should match:
+      | Successfully pulled image "image-registry.openshift-image-registry.svc:5000/<%= project.name %>/mysql | 
+    When I run the :import_image client command with:
+      | from       | <%= cb.mirror_registry %>rhscl/ruby-25-rhel7:latest |
+      | confirm    | true                                                |
```

Comment:
Align issue.

Reply:
It keeps same space with next line.

Reply:
Checked in another computer, the align is good.